### PR TITLE
Picks up Common w/ PoP Clock-Skew Sync

### PR DESCRIPTION
**Pull request by iambmelt**
Expands:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/issues/864
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/868

### CODE REVIEW

File: msal/src/main/java/com/microsoft/identity/client/internal/controllers/OperationParametersAdapter.java [hunk 1]
```diff
@@ -97,6 +97,7 @@ public static AcquireTokenOperationParameters createAcquireTokenOperationParamet
 
         acquireTokenOperationParameters.setAuthenticationScheme(
                 AuthenticationSchemeFactory.createScheme(
+                        acquireTokenParameters.getActivity(),
```
Comment (shahzaibj) [inline msal/src/main/java/com/microsoft/identity/client/internal/controllers/OperationParametersAdapter.java]: Is there a `createScheme` method that takes an `Activity`? I can't find it.

Comment (iambmelt) [reply msal/src/main/java/com/microsoft/identity/client/internal/controllers/OperationParametersAdapter.java]: `Activity` is a subclass of `Context`

Comment (iambmelt) [reply msal/src/main/java/com/microsoft/identity/client/internal/controllers/OperationParametersAdapter.java]: Btw I am going to clean up this constructor a little bit to remove the hard reference to the Context and instead include a reference to the `IClockSkewManger` -- will be pushed up momentarily

Comment (iambmelt) [reply msal/src/main/java/com/microsoft/identity/client/internal/controllers/OperationParametersAdapter.java]: Cleaned up here:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/868/commits/dff13326e369b599f0ea4aa483c401431b56076b

**Pull request merged**